PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!-- Open this PR as draft while it is not ready -->
+
+### Description
+
+<!-- Describe your solution for the issue here. Adding videos, screenshots and expected logs may help the others to understand what is new.
+-->
+
+### Related Issues
+<!--
+Use keywords like 'close' or 'solves' to link this PR to an issue.
+For example:
+
+Closes #12345
+Unblocks #54321
+-->
+
+### Progress
+
+<!-- If your PR is WIP, use checkboxes to show that you did and what you have to do. For example: -->
+
+- [x] New endpoint created;
+- [ ] Update organizations;
+- [ ] Create tests;
+
+This PR solves #12345
+
+#### Pull request checklist
+
+<!-- Before submitting the PR, please address each item -->
+
+- [ ] **Tests**: This PR includes tests for covering the features or bug fixes, or it includes an explanation of why it does not.
+- [ ] **Screenshots and/or logs**: This PR includes screenshots, videos or logs of the changes made, or an explanation of why it does not.
```

Comment:
There is a section  "_Visual Reference_" below, which I think is enough to remind the coder about adding the visual reference (and the reviewer can readily see whether there's one or not, so I'm not sure this check here is very useful).
And many issues don't ask for a visual reference (say, chore issues or refactor issues), so having to explain why you don't have it also seems a bit much.

Maybe you'd like to add an optional subsection for "_Change Logs_" in the "_Description_"? I think it should contain a bullet list summary of the changes

Reply:
Screenshots checkbox removed and `Change Logs` section created.